PowerPointLabs/PowerPointLabs/Utils/ShapeUtil.cs

```diff
@@ -107,6 +107,13 @@ public static bool CanAddArrows(Shape shape)
             }
         }
 
+        public static bool IsShapeOrLineOrTextBox(Shape shape)
```

Comment:
Maybe we can differ the refactoring for a bit.
Things get a bit complicated when we add support for Pictures & Placeholders @ synclab